### PR TITLE
Fix typo in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rq1995**
Fix typo in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
-#Contributing
+# Contributing
 
 Persons contributing to this project agree to the [code of conduct](./CONDUCT.md) and agree to the potential redistribution of their code under our license.
```
Comment (hadinh1306) [inline CONTRIBUTING.md]: (./CODE_OF_CONDUCT.md)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,12 +41,12 @@ Persons contributing to this project agree to the [code of conduct](./CONDUCT.md
...
-##How to Contribute
+## How to Contribute
 
-####Code comment 
+#### Code comment 
 - Functions need to be documented clearly.
 
-####Commit message
+#### Commit message
 - Big changes:
```
Comment (hadinh1306) [inline CONTRIBUTING.md]: Small changes:
```
git commit -m "Concise summary of what you did here"
```

Major changes:
```
git commit -m "Title of commit message
Details of your changes"
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
+- Write tests for function changes.
+- Once your changes are done, submit a Pull Request.
+- Wait for comments from team member. Then make changes from your Branch/Folk repo.
+- Only merge after reaching whole team agreement.
 
-##Where to Contribute(Workflow)
+## Where to Contribute(Workflow)
 
 #### Using GitHub
 1.  We can use GitHub flow (branch) to manage changes:
```
Comment (hadinh1306) [inline CONTRIBUTING.md]: Can you also add how to Folk in this section?

File: CONTRIBUTING.md (hunk 1 above)
Comment (rq1995) [reply CONTRIBUTING.md]: Thanks for reminding.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rq1995) [reply CONTRIBUTING.md]: We talk about it in the `How to contributing` part.

**Pull request merged**